### PR TITLE
fix(profile): apply ellipsis on long social link

### PR DESCRIPTION
**Pull request by idoshamun**
### Describe what this PR does

Long portfolio links broke the design of the page

Fixed version:
image

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/profile/SocialChips.tsx [hunk 1]
```diff
@@ -44,11 +44,11 @@ export function SocialChips({ links }: SocialChipsProps): ReactElement {
     .filter((key) => !!links[key])
     .map((key) => (
       <Button
-        spanClassName="w-fit my-2 font-normal"
+        spanClassName="my-2 font-normal text-ellipsis whitespace-nowrap overflow-hidden flex-1"
```
Comment (rebelchris) [inline packages/shared/src/components/profile/SocialChips.tsx:47]: Dang see this is passed through, else we could use `getTextEllipsis` function/component.

**Review by rebelchris — APPROVED** [review]
Thanks for the quick fix.

File: packages/shared/src/components/profile/SocialChips.tsx (hunk 1 above)
Comment (idoshamun) [reply packages/shared/src/components/profile/SocialChips.tsx:47]: Sorry I auto merged. But I see that `getTextEllipsis` returns a component which I can't use as I'm using the button element. With that said, I believe having it by default in the button element can be super beneficial

Comment (rebelchris) [reply packages/shared/src/components/profile/SocialChips.tsx:47]: Hmm yeah could be useful as the default on buttons indeed!

**Pull request merged**